test/optimizer/rules/test_rules.py

```diff
@@ -195,7 +211,9 @@ def test_supported_rules(self):
             LogicalInsertToPhysical(),
             LogicalDeleteToPhysical(),
             LogicalLoadToPhysical(),
-            LogicalGetToSeqScan(),
+            DistributedLogicalGetToSeqScan()
+            if ray_enabled
+            else SequentialLogicalGetToSeqScan(),
```

Comment:
Could we add a comment that indicates `DistributedLogicalGetToSeqScan()` and `SequentialLogicalGetToSeqScan()` are same with a brief justification?